src/plots/cartesian/line_grid_defaults.js

```diff
@@ -32,18 +32,40 @@ module.exports = function handleLineGridDefaults(containerIn, containerOut, coer
         delete containerOut.linewidth;
     }
 
-    var gridColorDflt = colorMix(dfltColor, opts.bgColor, opts.blend || lightFraction).toRgbString();
+    var gridColorDflt = colorMix(dfltColor, opts.bgColor, opts.blend || colorAttrs.lightFraction).toRgbString();
     var gridColor = coerce2('gridcolor', gridColorDflt);
     var gridWidth = coerce2('gridwidth');
     var gridDash = coerce2('griddash');
-    var showGridLines = coerce('showgrid', opts.showGrid || !!gridColor || !!gridWidth || !!gridDash);
+    var showGridLines = coerce('showgrid', opts.showGrid ||
+        !!gridColor ||
+        !!gridWidth ||
+        !!gridDash
+    );
 
     if(!showGridLines) {
         delete containerOut.gridcolor;
         delete containerOut.gridwidth;
         delete containerOut.griddash;
     }
 
+    if(opts.hasMinor) {
+        var minorGridColorDflt = colorMix(gridColor || gridColorDflt, opts.bgColor).toRgbString();
+        var minorGridColor = coerce2('minor.gridcolor', minorGridColorDflt);
+        var minorGridWidth = coerce2('minor.gridwidth');
+        var minorGridDash = coerce2('minor.griddash');
```

Comment:
Should these two inherit unchanged from the major gridlines?

Reply:
No. I don't think so.
By default minor ticks and grid lines are drawn to be quite as minimal as possible.
If one wants to emphasis on major ticks or major grid lines, then it could be as simple as changing the main attributes. Otherwise they have to figure out and reset minor values too which is not desirable.

Reply:
FWIW I agree with AJ here :)